languages/csharp/exercises/concept/numbers/.docs/hints.md

```diff
@@ -1,18 +1,20 @@
 ### General
 
-- [docs.microsoft.com numbers tutorial][tutorial-docs.microsoft-numbers]
+- [Numbers tutorial][numbers].
 
 ### 1. Calculate the production rate per second
 
-- Determining the success rate can be done through a [conditional statement][tutorial-csharp.net-if-statement].
+- Determining the success rate can be done through a [conditional statement][if-statement].
 - C# allows for multiplication to be applied to two different number types (such as an `int` and a `double`). It will automatically return the "largest" data type.
-
-[tutorial-docs.microsoft-numbers]: https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/intro-to-csharp/numbers-in-csharp-local
-[tutorial-csharp.net-if-statement]: https://csharp.net-tutorials.com/control-structures/if-statement/
+- Numbers can be compared using the built-in [comparison-][comparison-operators] and [equality operators][equality-operators].
 
 ### 2. Calculate the number of working items produced per second
 
-- Whereas an `int` can be automatically converted to a `double`, the reverse does not hold. The reason for this is that an `int` has less precision than a `double` so rounding has to be applied, also the range of numbers an `int` can represent is smaller than a `double`. To force this conversion, one can either use one of the [`Convert` class' methods][docs-microsoft.com-convert] or [cast to an int][tutorial-dotnetperls.com-cast-int].
+- Whereas an `int` can be automatically converted to a `double`, the reverse does not hold. The reason for this is that an `int` has less precision than a `double` so rounding has to be applied, also the range of numbers an `int` can represent is smaller than a `double`. To force this conversion, one can either use one of the [`Convert` class' methods][convert-class] or [cast to an int][cast-int].
 
-[docs-microsoft.com-convert]: https://docs.microsoft.com/en-us/dotnet/api/system.convert?view=netcore-3.0#examples
-[tutorial-dotnetperls.com-cast-int]: https://www.dotnetperls.com/cast-int
+[convert-class]: https://docs.microsoft.com/en-us/dotnet/api/system.convert?view=netcore-3.1#examples
+[cast-int]: https://www.dotnetperls.com/cast-int
+[numbers]: https://docs.microsoft.com/en-us/dotnet/csharp/tutorials/intro-to-csharp/numbers-in-csharp-local
+[if-statement]: https://csharp.net-tutorials.com/control-structures/if-statement/
+[comparison-operators]: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/comparison-operators
+[equality-operators]: https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/equality-operators
```

Comment:
Are these in the wrong document? Looks like they should be in `after.md` rather than `hints.md` given the current markdown in those documents

Reply:
Nope, they _are_ used in the `hints.md` document too.